### ISSUE TITLE
Please, checks after merge

### ISSUE THREAD

**Issue by Anton-Latukha** [haskell/actions#86]
There is a number of things that need to be done & checked to make merged `hlint` actions work properly.

  - [ ] 1. I forgot to manually update my [`haskell/actions`](https://github.com/Anton-Latukha/actions?organization=Anton-Latukha&organization=Anton-Latukha) main fork branch to recent commits, that is - after #82 & #83 got merged.

A big diff in #83 thankfully is only `haskell/actions/setup` specific. There everything is Ok, seems it requires nothing.

Also #82's `.github/dependabot.yml` is not in #85's Git history. During merge that would resolve itself. Again - that should be no biggy at all. But because I have already redone complex merge several times & then noticed:
```
hlint-run/.github/dependabot.yml
hlint-setup/.github/dependabot.yml
```
can not be merged with `.github/dependabot.yml` in the scope of #85, because the `.github/dependabot.yml` does not exist there.

After the merge into the mainline - configs of those `dependabot.yml` files seems need merge into `.github`. As I found no evidence/docs that localized `dependabot` configuration would work. Seems like setting according `directory: ` field there would be enough.

  - [ ] 2. (optional):
  
  I kept `.gitattributes` files in according directories. Documentation on `gitattributes` for me was unclear - does `git` treats them as it does `.gitignore` (allowing to place localized `.gitignore` in any directory. I currently assume, `git` detects & works with `.gitattributes` localized just as well.

  - [ ] 2. Tags for hlint action versioning need to be made (as GitHub Actions mechanism depends on that) (as the continuations of `rwe`s versioning). & as `alpha/beta/pre` first.

  - [ ] 3. HLint actions (their testing prereleases) need to be published to GitHub Marketplace, so we can test the full actions workflow loop.

  - [ ] 4. Final testing
  
I can test pre-release & then stable release actions on
  * [haskell-nix/hnix](https://github.com/haskell-nix/hnix)
  * [haskell-nix/hnix-store](https://github.com/haskell-nix/hnix-store) - monorepo.

And on more calm [haskell-with-nixpkgs](https://github.com/haskell-nix/haskell-with-nixpkgs) for that matter.

  - [ ] 5. Closes:
    
    * #81
    * https://github.com/rwe/actions-hlint-setup/issues/135
    * https://github.com/rwe/actions-hlint-run/issues/137
    * Allows to solve https://github.com/haskell/haskell-language-server/issues/2143.

**Comment by jared-w** [comment]
Sweet, this should be working now. One thing to note is that since it's a monorepo, the versioning used by github actions applies uniformly to all of the actions. As such, a breaking change that would require a v2 would mean they're all now v2; due to the nature of these actions (they don't do complicated things), I'm not particularly worried about that, as it shouldn't be difficult to keep the api backwards compatible for them indefinitely.

**Comment by Anton-Latukha** [comment]
I think the merge is comfortable in `v1.0` bounds. The old `hlint` action versioning does not need to be preserved when it essentially changes the name (address).

After I do a couple of things here & there, would do the finishing touches on this merging procedure.

Then the minor dot release would be asked to do, just to keep old git tag in place & create a new one with the new actions.

**Comment by andreasabel** [comment]
Not sure why this issue is still open.  Closing...

> Sweet, this should be working now. One thing to note is that since it's a monorepo, the versioning used by github actions applies uniformly to all of the actions. As such, a breaking change that would require a v2 would mean they're all now v2; due to the nature of these actions (they don't do complicated things), I'm not particularly worried about that, as it shouldn't be difficult to keep the api backwards compatible for them indefinitely.

On that, see:
- #141
